conf/setup.sql

```diff
@@ -1,5 +1,5 @@
 # Create the DB
-CREATE DATABASE css;
+CREATE DATABASE css character set UTF8mb4 collate utf8mb4_bin;
```

Comment:
I'm a little worried about conflicting settings in the database and the settings you're applying to a single database.  Your extending the amount of data and symbols the database can hold I get that, but I'm thinking that without setting certain settings in the mysql conf could cause compatibility issues.  Please verify that it does not or add something to the bootstrap that does it, to verify you can run `SHOW VARIABLES WHERE Variable_name LIKE 'character\_set\_%' OR Variable_name LIKE 'collation%';`